README.md

```diff
@@ -1,3 +1,54 @@
 # register-country-by-country-reporting-frontend
 
-This is a placeholder README.md for a new repository
\ No newline at end of file
+## Info
+
+This service allows a user(agent, client or organisation) to register to CBCR services.
+
+This service has a corresponding back-end service, namely register-country-by-country-reporting can be viewed [here](https://github.com/hmrc/register-country-by-country-reporting) which integrates with HOD, i.e. DES/ETMP.
+
+### Dependencies
+
+| Service                                              | Link                                                                    |
+|------------------------------------------------------|-------------------------------------------------------------------------| 
+| Address Lookup                                       | https://github.com/hmrc/address-lookup                                  |
+| Email                                                | https://github.com/hmrc/email                                           |
+| Auth                                                 | https://github.com/hmrc/auth                                            |
+| Tax Enrolments                                       | https://github.com/hmrc/tax-enrolments                                  |
+| Country by country reporting frontend                | https://github.com/hmrc/country-by-country-reporting-frontend           |
+| Country by country reporting                         | https://github.com/hmrc/country-by-country-reporting-frontend           |
+| Country by country reporting stubs                   | https://github.com/hmrc/country-by-country-reporting-stubs              |
+| Register country by country reporting frontend stubs | https://github.com/hmrc/register-country-by-country-reporting-stubs     |
```

Comment:
Register country-by-country-frontend and register-country-by-country-stubs are 2 different services. You only need to call stubs here.

Reply:
Sorry got it now ,  fixed it.